### DISCUSSION TITLE
code.europa.eu

### DISCUSSION THREAD

**Discussion by black-snow** [OP-TED/eForms-SDK#152] [Q&A] [answered]
Out of curiousity, will this project move to https://code.europa.eu ? Seems to be a more obvious choice now than US-based GitHub.

**Comment by bertrand-lorentz** [comment, marked as answer]
Hello,
I've moved this to "Discussions", as it's more suited for this type of question.

The website https://code.europa.eu did not exist when we started publishing the eForms SDK, back in March 2021.
We currently have no plans to migrate to it. With the coming deadlines for the transition to eForms, we prefer to avoid any disruption that such a migration could cause.
We will have a closer look into our options after the transition to eForms is completed.

**Reply by black-snow** [reply]
Aight, thanks for the reply.
